<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>netMusic1</class>
 <widget class="QWidget" name="netMusic1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>290</width>
     <height>372</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblLunbo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>1151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>50</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rBtn">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>80</y>
     <width>50</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>290</width>
     <height>372</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>290</width>
     <height>372</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>390</y>
     <width>290</width>
     <height>372</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnGedan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
